arangod/Cache/Cache.cpp

```diff
@@ -343,10 +343,7 @@ void Cache::shutdown() {
       }
 
       SpinUnlocker taskUnguard(SpinUnlocker::Mode::Write, _taskLock);
-      SpinUnlocker shutdownUnguard(SpinUnlocker::Mode::Write, _shutdownLock);
-
-      // sleep a bit without holding the locks
-      std::this_thread::sleep_for(std::chrono::microseconds(20));
```

Comment:
Hmm. why did you change it in reverted commit?
Because additional lock?

Reply:
I reverted the entire previous commit.
The version before had known issues, but these were probably less severe than the issues introduced by the commit that is now reverted.

Reply:
I understand but just curios why in reverted version you decide to increase sleep_for time